core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -462,7 +462,12 @@ object KafkaConfig {
     "the same security protocol to be usable in more than one port or IP. For example, we can separate internal and " +
     "external traffic even if SSL is required for both. Concretely, we could define listeners with names INTERNAL " +
     "and EXTERNAL and this property as: `INTERNAL:SSL,EXTERNAL:SSL`. As shown, key and value are separated by a colon " +
-    "and map entries are separated by commas. Each listener name should only appear once in the map."
+    "and map entries are separated by commas. Each listener name should only appear once in the map. " +
+    "We can provide different security (SSL and SASL) settings for each listener name by adding a normalised " +
+    "prefix (the listener name is lowercased) to the config name. For example, if we wanted to set a different " +
+    "keystore for the INTERNAL listener, we would set a config with name listener.name.internal.ssl.keystore.location. " +
+    "If the config for the listener name is not set, we will fallback to the generic config (i.e. ssl.keystore.location). " +
+    "See KIP-103 for more details."
```

Comment:
I don't think we reference KIPs for any other config doc. A KIP isn't really the place for documenting the current implementation, since no one maintains KIPs.

Reply:
Yes, we are not referring KIPs in config descriptions. but we are referring them in general documentation. 
KIP reference may not be necessary here. Updated the PR

Reply:
@ijuma  Please review this.